### DISCUSSION TITLE
Swup and Lenis Scroll

### DISCUSSION THREAD

**Discussion by DanielKNDZ** [swup/swup#520] [Help]
Hello. Im using Swup with Lenis Scroll (scroll smoothness). The problem is when i'm scrolling page A and go to the page B (right at the moment of page A scrolling animation) page B is scrolling too. I've used `window.scrollTo(0, 0);` and it works fine, but only when the scroll animation is fully stopped on page A. Any ideas how to solve this problem?
_Sorry for my English, not native_

**Lenis code**
[https://codepen.io/dackjaniels/pen/ExRYJYd](url)

**Swup**

`const swup = new Swup(); 
swup.on('contentReplaced', function () {
window.scrollTo(0, 0);
});`

**Comment by daun** [comment]
The CodePen is empty :(
